src/store/getters.ts

```diff
@@ -785,7 +781,7 @@ export const getNameAnalysisTimeout = (state: StateIF): boolean => {
 }
 
 export const getPriorityRequest = (state: StateIF): boolean => {
-  return state.stateModel.newRequestModel.priorityRequest
+  return state.stateModel.newRequestModel.priorityRequest || false
```

Comment:
How does this boolean relate to the `priorityRequestCd`?
I think i missed the context use of the code.

Reply:
Internally we use `priorityRequest`, which is a boolean.

Externally (ie, to/from API), we use priorityRequestCd, which is either Y or N.

I wasn't trying to refactor everything, just some cleanup on code I touched.